website/docs/docs/build/python-models.md

```diff
@@ -216,7 +216,79 @@ def model(dbt, session):
 
 ### Materializations
 
-Python models support dbt Materializations.  To learn more about them visit the [Materializations page](/docs/build/materializations.md)
+Python models support these materializations:
+- `table`
+- `incremental`
+
+Incremental Python models support all the same [incremental strategies](/docs/build/incremental-models#about-incremental_strategy) as their SQL counterparts. The specific strategies supported depend on your adapter.
+
+Python models can't be materialized as `view` or `ephemeral`. Python isn't supported for non-model resource types (like tests and snapshots).
+
+For incremental models, like SQL models, you need to filter incoming tables to only new rows of data:
+
+<WHCode>
+
+<div warehouse="Snowpark">
+
+<File name='models/my_python_model.py'>
+
+```python
+import snowflake.snowpark.functions as F
+
+def model(dbt, session):
+    dbt.config(materialized = "incremental")
+    df = dbt.ref("upstream_table")
+
+    if dbt.is_incremental:
+
+        # only new rows compared to max in current table
+        max_from_this = f"select max(updated_at) from {dbt.this}"
+        df = df.filter(df.updated_at >= session.sql(max_from_this).collect()[0][0])
+
+        # or only rows from the past 3 days
+        df = df.filter(df.updated_at >= F.dateadd("day", F.lit(-3), F.current_timestamp()))
+
+    ...
+
+    return df
+```
+
+</File>
+
+</div>
+
+<div warehouse="PySpark">
+
+<File name='models/my_python_model.py'>
+
+```python
+import pyspark.sql.functions as F
+
+def model(dbt, session):
+    dbt.config(materialized = "incremental")
+    df = dbt.ref("upstream_table")
+
+    if dbt.is_incremental:
+
+        # only new rows compared to max in current table
+        max_from_this = f"select max(updated_at) from {dbt.this}"
+        df = df.filter(df.updated_at >= session.sql(max_from_this).collect()[0][0])
+
+        # or only rows from the past 3 days
+        df = df.filter(df.updated_at >= F.date_add(F.current_timestamp(), F.lit(-3)))
+
+    ...
+
+    return df
+```
+
+</File>
+
+</div>
+
+</WHCode>
+
+Incremental models are supported on BigQuery with Dataproc for the `merge` incremental strategy. The `insert_overwrite` strategy is not yet supported.
```

Comment:
Kind of feel like this should be up at line 223 in the form of a disclaimer of some sort. Not sure the best way to format that with our system but having this information located so far away from that statement could lead to confusion.

Reply:
thanks for flagging!